src/client/components/HierarchyContainer/HierarchyContainer.component.js

```diff
@@ -84,13 +84,13 @@ function getRenderedTopics(topics) {
 
   let rendered = null;
 
-  if (topics.length > 5) {
+  if (topics.length > 7) {
     rendered = (
       <div>
         <HierarchyElementTopics topics={topics.slice(0, 5)}/>
         <ToggleContainer show={false}>
           <ToggleContent content={<HierarchyElementTopics topics={topics.slice(5)}/>}/>
-          <ToggleButton showText={`Vis flere (${topics.slice(5).length})`} hideText='Skjul'/>
+          <ToggleButton showText={`Vis alle (${topics.slice(5).length})`} hideText='Skjul'/>
```

Comment:
Nej, tanken var at lave cuttet ved 5 men kun hvis der som minimum er 3 mere at vise. Hvis der er en enkelt eller to kan man lige så godt vise dem med det samme i stedet for at lave en drop down til kun en eller to.